### DISCUSSION TITLE
Copilot: Disabled for filetype= by internal default

### DISCUSSION THREAD

**Discussion by adri1n** [community/community#63594] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm getting the following error`Copilot: Disabled for filetype= by internal default` when I do `:Copilot status` on a `.py` file. As you can imagine, copilot autocompletion does not work at all for any file. How can I solve this? I am working with vim.

**Comment by Akash1134** [comment]
Hi @adri1n ,

We're sorry about the hassle. Which IDE are you using and what is the version of Copilot you have installed?

Have you tried logging out of Copilot and logged back in? Are you connected to proxy network and does this issue occur every time regardless of which network you are connected to? 

Thanks!

**Comment by shaunemslie** [comment, marked as answer]
Hi @adri1n,

I've seen similar issues occur however the filetype would be included in the error, e.g `filetype=markdown`
So, logically it seems to me that your filetypes, specifically for python, aren't being set. If you try the command `:set filetype` inside a `.py` file what is the command response?

Additionally, do you have `filetype on` set in your `~/.vimrc`?

**Reply by adri1n** [reply]
Thanks @shaunemslie! Setting `filetype on` in `~/.vimrc` fixed the issue for all files! `set filetype=python` is also useful.

**Reply by shaunemslie** [reply]
Glad it's all sorted! 😁 Cheers.
